Code/connect/src/Server.cpp

```diff
@@ -34,13 +34,18 @@ namespace TiltedPhoques
         SteamInterface::Release();
     }
 
-    bool Server::Host(const uint16_t aPort, uint32_t aTickRate) noexcept
+    bool Server::Host(const uint16_t aPort, uint32_t aTickRate, const bool bEnableDualStackIP) noexcept
     {
         Close();
 
         SteamNetworkingIPAddr localAddress{};  // NOLINT(cppcoreguidelines-pro-type-member-init)
-        localAddress.Clear();
-        localAddress.m_port = aPort;
+        if (bEnableDualStackIP == true) {
+            localAddress.Clear();
+            localAddress.m_port = aPort;
+        }

```

Comment:
Please adjust your indent according to our coding standards. E.g. braces only on a newline.

Reply:
Will do, but isn't this something that should be enforced with a pre-commit hook? (I could make a PR for this I guess)